src/components/Article.jsx

```diff
@@ -1 +1,42 @@
 // export const Article = () => ();
```

Comment:
don't forget to delete redundant comments

Reply:
agree